test/.eslintrc

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
This file can be moved into the `mocha` folder.  This is because the `env.mocha` specified here is only applicable to the test spec files.  Also, this is how the template module is laid out:

https://github.com/digitalbazaar/bedrock-module-template-http/tree/main/test/mocha